packages/cli/__tests__/zosfiles/__unit__/upload/ftds/FileToDataSet.handler.unit.test.ts

```diff
@@ -329,8 +329,9 @@ describe("Upload file-to-data-set handler", () => {
                     statusMessage: "Uploading to data set"
                 }
             });
-            expect(jsonObj).toMatchSnapshot();
-            expect(apiMessage).toMatchSnapshot();
+
+            expect(apiMessage).toBe("");
+
```

Comment:
While I support removing snapshots when it makes sense, I think it's also important not to reduce the quality of our tests when doing so. Should we keep an `expect` statement in this test to validate the contents of the JSON object?

Reply:
Please disregard this comment - after discussing with @zFernand0 it makes sense to remove this snapshot because the JSON object comes from a mock earlier in the test 😋